pom.xml

```diff
@@ -130,6 +136,10 @@
       <groupId>org.eclipse.jetty.http2</groupId>
       <artifactId>jetty-http2-server</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty.websocket</groupId>
+      <artifactId>jetty-websocket-jetty-api</artifactId>
+    </dependency>
```

Comment:
Needed to implement WebSocket agent support in https://github.com/jenkinsci/jenkins/pull/10461.